multi-educator/src/Dscp/MultiEducator/Types.hs

```diff
@@ -0,0 +1,34 @@
+module Dscp.MultiEducator.Types
+    ( EducatorId (..)
+    ) where
+
+import Control.Lens ((.=))
+import Data.Aeson (FromJSON (..), ToJSON (..))
+import qualified Data.Swagger as S
+import Fmt (build)
+import Servant (FromHttpApiData (..), ToHttpApiData (..))
+
+import Dscp.Util
+import Dscp.Web.Swagger
+
+-- TODO: this type name is a duplicate, do something
+newtype EducatorId = EducatorId Text
+    deriving (Show, Eq, Ord, IsString)
```

Comment:
Maybe, import the original?

Reply:
I'd like to, but they are different inside `Address` vs `Text`, want someone to explain me how it happened :smile:

Reply:
Oh, I just found out that the original it not even used anywhere :peka: :paket: